sdk/src/loader_instruction.rs

```diff
@@ -15,6 +16,7 @@ pub enum LoaderInstruction {
     /// bit of the Account
     ///
     /// * key[0] - the account to prepare for execution
+    /// * key[1] - rent sysvar account
```

Comment:
@ParthDesai - how's your Javascript?  We'll need to update https://github.com/solana-labs/solana-web3.js/blob/b9841a302c577880f325a2a8bec6d31334d1f6a6/src/loader.js#L126 to include rent sysvar before/when this PR lands to keep the examples working.    It looks like we could update web3.js before and it would be no harm.